README.md

```diff
@@ -241,7 +241,7 @@ docker run --rm --name eosio_notechain_container \
 --mount type=bind,src="$(pwd)"/contracts,dst=/opt/eosio/bin/contracts \
 --mount type=bind,src="$(pwd)"/scripts,dst=/opt/eosio/bin/scripts \
 --mount type=bind,src="$(pwd)"/data,dst=/mnt/dev/data \
--w "/opt/eosio/bin/" eosio-notechain:eos1.7.0-cdt1.5.0 /bin/bash -c "./scripts/init_blockchain.sh"
+-w "/opt/eosio/bin/" eosio-notechain:eos1.8.0-cdt1.6.1 /bin/bash -c "./scripts/init_blockchain.sh"
```

Comment:
Can you drop the specific version reference for the eosio-notechain Docker container?  The latest version would be used in that case, right?

Reply:
Right, if the specific version reference tag is dropped, it will always use the latest version (eosio-notechain:latest). However, if you take a look at first_time_setup.sh, the docker image will always be created with a specific tag. So eosio-notechain:latest will never be created.

We always tag the image with a specific version, so we can skip building the image if there's nothing changed every time the user calls ./quick_start.sh (which saves the user a few seconds every time).

Reply:
The version tag is added to modifying the docker image name to ensure user would create a new one.